src/net.cpp

```diff
@@ -1391,6 +1398,42 @@ void MapPort(bool)
 }
 #endif
 
+// BITCOINUNLIMITED START
+void ThreadBitnodesAddressSeed()
+{
+    // Get nodes from websites offering Bitnodes API
+    if ((addrman.size() > 0) &&
+        (!GetBoolArg("-forcebitnodes", DEFAULT_FORCEBITNODES))) {
+        MilliSleep(11 * 1000);
```

Comment:
It's a magic number.  For consistency I used the same value as found in ThreadDNSAddressSeed().

https://github.com/BitcoinUnlimited/BitcoinUnlimited/blob/0.12bu/src/net.cpp#L1400